docs/en/latest/plugins/file-logger.md

```diff
@@ -31,6 +31,16 @@ description: This document contains information about the Apache APISIX file-log
 
 The `file-logger` Plugin is used to push log streams to a specific location.
 
+:::tip
+
+- `file-logger` plugin can count request and response data for individual routes locally, which is useful for [debugging](../debug-mode.md).
+- `file-logger` plugin can get [APISIX variables](../apisix-variable.md) and [NGINX variables](http://nginx.org/en/docs/varindex.html), while `access.log` can only use NGINX variables.
+- `file-logger` plugin support hot-loaded so that we can change its configuration at any time with immediate effect.
+- `file-logger` plugin saves every data in JSON format.
+- The user can modify the functions executed by the `file-logger` during the `log phase` to collect the information they want.
+
+:::
+
```

Comment:
Reference Chinese modification

Reply:
Hi, I recommend English to stay the same.